pytest_cafy/plugin.py

```diff
@@ -1034,6 +1027,37 @@ def pytest_runtest_logreport(self, report):
                     else:
                         self.testcase_failtrace_dict[testcase_name] = None
 
+    @pytest.hookimpl(hookwrapper=True, trylast=True)
+    def pytest_runtest_makereport(self, item, call):
+        print("in cafy plugin makereport")
```

Comment:
Is this print intentional or was it for testing?

Reply:
I will remove it